db/operations.go

```diff
@@ -151,8 +160,33 @@ func formatValue(tableName, columnName, value string, valueType reflect.Type) (s
 			v := time.Unix(int64(i), 0).Format(time.RFC3339)
 			return fmt.Sprintf("'%s'", v), nil
 		}
-		return "", fmt.Errorf("unsupported type %s for column %s in table %s", valueType, columnName, tableName)
+		return "", fmt.Errorf("unsupported struct type %s", valueType)
 	default:
-		return "", fmt.Errorf("unsupported type %s for column %s in table %s", valueType, columnName, tableName)
+		return "", fmt.Errorf("unsupported type %s", valueType)
 	}
 }
+
+var charsToEscape = []string{"\\", "\n", "\r", "\t", `'`, `"`, "\b", "\f", "%", "_"}
+
+func escapeString(valueToEscape string, escapeType string) string {
+	escaped := valueToEscape
+
+	if escapeType == "column" {
+		if strings.Contains(valueToEscape, `"`) {
+			escaped = strings.ReplaceAll(valueToEscape, `"`, `""`)
+		}
+		escaped = fmt.Sprintf(`"%s"`, escaped)
+
+		if reservedKeywords[strings.ToUpper(valueToEscape)] {
+			escaped = fmt.Sprintf(`"%s"`, valueToEscape)
+		}
+		return escaped
```

Comment:
Does the same thing in all cases. Also for such short pattern, it's preferable to concatenate the string directly.
```suggestion
		return `"` + escaped + `"`
```